geopandas/tools/tests/test_sjoin.py

```diff
@@ -87,10 +86,10 @@ def dfs(request):
 
 
 @pytest.mark.parametrize("dfs", ["default-index"], indirect=True)
-def test_raises_error_if_rtree_not_install(dfs):
```

Comment:
We may not need this test anymore, since the check is done elsewhere.

Reply:
I'm +1 for removing it, but I didn't want to delete any tests unless others agree.

Reply:
Indeed this is being tested in `sindex` now, and since the whole point of this PR is to move the burden or error handling to `sindex`, this test _should_ be removed. I pushed a commit that does so.